.internal-ci/helm/fog-ingest-config/values.yaml

```diff
@@ -27,3 +27,6 @@ fogIngest:
   POSTGRES_CONNECTION_TIMEOUT: '5'
   POSTGRES_MAX_CONNECTIONS: '3'
   FOG_PUBKEY_EXPIRY_WINDOW: '10'
+  # Bootstrap ledger/watcher db from public bucket
+  # MC_LEDGER_DB_URL: ''
+  # MC_WATCHER_DB_URL: ''
```

Comment:
It may be how you all are evolving the helm stuff, but in general I lean towards not committing commented out code, and not adding commented out code. 
```suggestion
```

Reply:
Normally I would agree, but the values.yaml file tends to serve as documentation and examples.  See something like https://github.com/goharbor/harbor-helm/blob/master/values.yaml on how the values are commented and explained.